### ISSUE TITLE
Show translation 

### ISSUE THREAD

**Issue by edlemus** [TeselaGen/openVectorEditor#836]
@tnrich
Hi again,
Is there a way to show the translations on the alignment tool?  (not cds feature translations, just translation of the sequence)
I can't find it on the documentation
<img width="474" alt="translation" src="https://user-images.githubusercontent.com/69165851/174654628-9673c2cf-9692-4386-bdf0-b3582b2354a6.PNG">

btw I'm using Vanilla JS
Thanks,

**Comment by tnrich** [comment]
@edlemus just checking, are you asking for a view of the Full sequence translation for all of the lanes?
Aka these options: 
<img width="1218" alt="image" src="https://user-images.githubusercontent.com/2730609/174673890-b8f89780-c938-4749-917b-614759d306f8.png">

but for the alignment view and for each individual lane?

I think that can be done but it might be a bit laggy potentially.

**Comment by edlemus** [comment]
Yes, that is correct. Full sequence translation for all of the lanes but for the alignment view

Ohh I see, hmm so that option for the alignment view is not available at the moment?

**Comment by tnrich** [comment]
@edlemus that option has not yet been added to the alignment view and would require additional work to add it. Not sure if you're interested in trying to PR that functionality

**Comment by edlemus** [comment]
@tnrich I'll give it a try

**Comment by tnrich** [comment]
@edlemus how's it going? :)

**Comment by edlemus** [comment]
@tnrich sorry, I've been busy with work. I'll get to it in a few days
